.travis.yml

```diff
@@ -16,6 +16,7 @@ env:
 before_install:
     - git clone --branch master --depth 1 https://github.com/Ensembl/ensembl-test.git
     - git clone --branch master --depth 1 https://github.com/Ensembl/ensembl.git
+    - git clone --branch master --depth 1 https://github.com/Ensembl/ensembl-hdf5.git
```

Comment:
if hdf5 is needed for the tests, it should also be added to the perl5lib in the harness